src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/RenderData.cs

```diff
@@ -52,6 +53,11 @@ internal RenderData()
             CanBeInheritanceContext = false;
         }
 
+        ~RenderData()
+        {
+            ArrayPool<byte>.Shared.Return(_buffer);
```

Comment:
Can this just be "disposed" normally? Using a finalizer is quite heavy and puts pressure on the GC's finalizer queue. Especially with 

> The max length is 72

in mind, maybe it's just cheaper to allocate and let the GC clean up. 
Or as there is unsafe code already involved use a struct with fixed sized buffer of that size, and wrap by a span for access outside that class?

Reply:
@gfoidl  Thank you and I update the PR